### PR TITLE
Final touches

### PR DESCRIPTION
**Pull request by Klumpish**
group coding fixed some smaller things with the homepage style and setup eventlistener for open modual button

### CODE REVIEW

File: challenges.js [hunk 1]
```diff
@@ -165,7 +165,7 @@ const challengeSort = {
 				btnDiv.classList.add("card__link");
 
 				/* Create button and append it to a div*/
-				console.log(this.ratingArray[i].id)
+				console.log(this.ratingArray[i].id);
```
Comment (marcusmoller97) [inline challenges.js:168]: This can be removed

File: js/booking.js [hunk 2]
```diff
@@ -16,40 +16,43 @@ const challengeInput = multiStepForm.querySelector("#challenge");
 const header1 = multiStepForm.querySelector("#header1");
 const header2 = multiStepForm.querySelector("#header2");
 
+import { ApiArray as events } from "../main.js";
 
-import {ApiArray as events} from "../main.js";
+// global variable to get id from challenge button.
+let eventId;
+// goes through all classes of .open-button and gets rooms number."
+openModal.forEach((openModal) =>
```
Comment (marcusmoller97) [inline js/booking.js:24]: Have we fixed so this works when using filter?

**Review by marcusmoller97 — COMMENTED** [review]
It looks good. Don't know if we fixed booking.js to work when we are filtering by for example tags.

File: js/booking.js (hunk 2 above)
Comment (Klumpish) [reply js/booking.js:24]: no not yet im working on it tho.

**Pull request merged**